samples/solution-counters.js

```diff
@@ -16,15 +16,15 @@
 const {Firestore, FieldValue} = require('@google-cloud/firestore');
 
 async function main() {
-  // [START increment_counter]
+  // [START firestore_solution_sharded_counter_increment]
```

Comment:
similar to the other document, can we keep boht tags for a bit? this aids the transition. 

```
// [START add_lovelace]
// [START firestore_setup_dataset_pt1]


/* SAMPLE CODE */

// [END firestore_setup_dataset_pt1]
// [END add_lovelace]
```

Reply:
Done!!